### ISSUE TITLE
Accessibility-> "Search by name or community" field is missing the form label

### ISSUE THREAD

**Issue by manuji** [bcgov/nr-rec-resources#590]
**Describe the bug**
The search field "Search by name or community" on the home page and Find a site or trail page are missing the form labels. This issue fails the [3.3.2 Labels or Instructions (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions.html) criterion. 

**To Reproduce**
1. Navigate to https://d1vrz2gt4nmfe5.cloudfront.net/
2. On Home page of the website check if the "Search by name or community" field has a form label.
3. Navigate to the Find a site or trail page and check if the "Search by name or community" field has a form label

**Expected behavior**
If a text label for a form control is visible, use the <label> element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using aria-labelledby. Labels are not required for image, submit, reset, button, or hidden form controls.

**Screenshots**

<img width="1422" alt="Image" src="https://github.com/user-attachments/assets/33e9cc1e-d498-4f78-9579-8d4bfca704bc" />

<img width="1392" alt="Image" src="https://github.com/user-attachments/assets/a6e88023-fb97-40fc-9cfc-392d47e7bba8" />

**Comment by airliah** [comment]
@manuji you can also take a look if/when you have time: https://d1vrz2gt4nmfe5.cloudfront.net/

**Comment by manuji** [comment]
Tested on Beta: Passed
* Verified that this issue has been fixed. 
* Moving to Done.
